Utilities/HairFlow/flow.js

```diff
@@ -20,70 +20,70 @@ Script.include(Script.resolvePath("./VectorMath.js"));
     var SHOW_DEBUG_SHAPES = false;
     var SHOW_SOLID_SHAPES = false;
     var SHOW_DUMMY_JOINTS = false;
-    var USE_COLLISIONS = false;
-    
+    var USE_COLLISIONS = true;
```

Comment:
making sure - did you mean to switch this bool?

Reply:
Yeah, I though it was better to activate collisions by default. I also changed some default params.